Cubical/Foundations/Equiv.agda

```diff
@@ -174,15 +183,23 @@ equivImplicitΠCod k .snd .equiv-proof f .snd (g , p) i .fst {x} =
 equivImplicitΠCod k .snd .equiv-proof f .snd (g , p) i .snd j {x} =
   equivCtrPath (k {x}) (f {x}) (g {x} , λ k → p k {x}) i .snd j
 
+equiv→Iso : (A ≃ B) → (C ≃ D) → Iso (A → C) (B → D)
+equiv→Iso h k .Iso.fun f b = equivFun k (f (invEq h b))
+equiv→Iso h k .Iso.inv g a = invEq k (g (equivFun h a))
+equiv→Iso h k .Iso.rightInv g = funExt λ b → retEq k _ ∙ cong g (retEq h b)
+equiv→Iso h k .Iso.leftInv f = funExt λ a → secEq k _ ∙ cong f (secEq h a)
+
 equiv→ : (A ≃ B) → (C ≃ D) → (A → C) ≃ (B → D)
-equiv→ h k = isoToEquiv isom
-  where
-  open Iso
-  isom : Iso _ _
-  isom .fun f b = equivFun k (f (invEq h b))
-  isom .inv g a = invEq k (g (equivFun h a))
-  isom .rightInv g = funExt λ b → retEq k _ ∙ cong g (retEq h b)
-  isom .leftInv f = funExt λ a → secEq k _ ∙ cong f (secEq h a)
+equiv→ h k = isoToEquiv (equiv→Iso h k)
+
+equivCompIso : (A ≃ B) → (C ≃ D) → Iso (A ≃ C) (B ≃ D)
+equivCompIso h k .Iso.fun f = compEquiv (compEquiv (invEquiv h) f) k
+equivCompIso h k .Iso.inv g = compEquiv (compEquiv h g) (invEquiv k)
+equivCompIso h k .Iso.rightInv g = equivEq _ _ (equiv→Iso h k .Iso.rightInv (equivFun g))
+equivCompIso h k .Iso.leftInv f = equivEq _ _ (equiv→Iso h k .Iso.leftInv (equivFun f))
+
+equivComp : (A ≃ B) → (C ≃ D) → (A ≃ C) ≃ (B ≃ D)
+equivComp h k = isoToEquiv (equivCompIso h k)
```

Comment:
I really wonder if one needs to go back and forth this much between equivalences and isos... Can't some of the proofs be done more directly?